src/test/java/com/commercetools/sync/cartdiscounts/utils/CartDiscountUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,563 @@
+    }
+
+    @Test
+    public void buildChangeIsActiveUpdateAction_WithSameValues_ShouldNotBuildUpdateAction() {
+        final CartDiscountDraft newCartDiscountDraft = mock(CartDiscountDraft.class);
+        when(newCartDiscountDraft.isActive()).thenReturn(false);
+
+        final Optional<UpdateAction<CartDiscount>> changeIsActiveUpdateAction =
+            buildChangeIsActiveUpdateAction(CART_DISCOUNT_WITH_RELATIVE_VALUE, newCartDiscountDraft);
+
+        assertThat(changeIsActiveUpdateAction).isNotPresent();
+    }
+
+    @Test
+    public void buildChangeNameUpdateAction_WithDifferentValues_ShouldBuildUpdateAction() {
+        final CartDiscountDraft newCartDiscountDraft = mock(CartDiscountDraft.class);
+        when(newCartDiscountDraft.getName()).thenReturn(LocalizedString.of(Locale.ENGLISH, "newName"));
+
+        final Optional<UpdateAction<CartDiscount>> changeNameUpdateAction =
+            buildChangeNameUpdateAction(CART_DISCOUNT_WITH_RELATIVE_VALUE, newCartDiscountDraft);
+
+        assertThat(changeNameUpdateAction).contains(ChangeName.of(LocalizedString.of(Locale.ENGLISH, "newName")));
+    }
+
+    @Test
+    public void buildChangeNameUpdateAction_WithNullValues_ShouldBuildUpdateAction() {
+        final CartDiscountDraft newCartDiscountDraft = mock(CartDiscountDraft.class);
+        when(newCartDiscountDraft.getName()).thenReturn(null);
+
+        final Optional<UpdateAction<CartDiscount>> changeNameUpdateAction =
+            buildChangeNameUpdateAction(CART_DISCOUNT_WITH_RELATIVE_VALUE, newCartDiscountDraft);
+
+        assertThat(changeNameUpdateAction).contains(ChangeName.of(null));
+    }
+
+    @Test
+    public void buildChangeNameUpdateAction_WithSameValues_ShouldNotBuildUpdateAction() {
+        final CartDiscountDraft newCartDiscountDraft = mock(CartDiscountDraft.class);
+        when(newCartDiscountDraft.getName()).thenReturn(LocalizedString.of(Locale.ENGLISH, "cart-discount-1"));
+
+        final Optional<UpdateAction<CartDiscount>> changeNameUpdateAction =
+            buildChangeNameUpdateAction(CART_DISCOUNT_WITH_RELATIVE_VALUE, newCartDiscountDraft);
+
+        assertThat(changeNameUpdateAction).isNotPresent();
+    }
+
+
+    @Test
+    public void buildSetDescriptionUpdateAction_WithDifferentValues_ShouldBuildUpdateAction() {
+        final CartDiscountDraft newCartDiscountDraft = mock(CartDiscountDraft.class);
+        when(newCartDiscountDraft.getDescription())
+            .thenReturn(LocalizedString.of(Locale.ENGLISH, "new-description"));
+
+        final Optional<UpdateAction<CartDiscount>> setDescriptionUpdateAction =
+            buildSetDescriptionUpdateAction(CART_DISCOUNT_WITH_RELATIVE_VALUE, newCartDiscountDraft);
+
+        assertThat(setDescriptionUpdateAction)
+            .contains(SetDescription.of(LocalizedString.of(Locale.ENGLISH, "new-description")));
+    }
+
+    @Test
+    public void buildSetDescriptionUpdateAction_WithNullValues_ShouldBuildUpdateAction() {
```

Comment:
like all the comments above: 

What about tests for:

both null values (old and new)?
just old null value?